curriculum/challenges/english/14-responsive-web-design-22/learn-css-variables-by-building-a-city-skyline/5d822fd413a79914d39e98f0.md

```diff
@@ -32,7 +32,7 @@ You should give the `background` a `linear-gradient`.
 assert.include(new __helpers.CSSHelp(document).getStyle('.bb1a')?.background, 'linear-gradient');
 ```
 
-You should give the `background` a `linear-gradient` starting from `--building-color1`.
+You should give the `background` a `linear-gradient` starting from `--building-color1` variable.
```

Comment:
I do not see this as needed

Reply:
@jeremylt It was being discussed in the issue as being confusing. The other alternative is 
```suggestion
You should give the `background` a `linear-gradient` starting from `var(--building-color1)`.
```

Reply:
The current seems grammatically awkward.

```suggestion
You should give the `background` a `linear-gradient` that starts with the `--building-color1` variable.
```

Maybe?

Reply:
The issue is still labeled discussion, so we can discuss what to do about it there.